demisto_sdk/commands/upload/uploader.py

```diff
@@ -257,6 +262,27 @@ def pack_uploader(self, path: str) -> int:
                 status_code = self.entity_dir_uploader(entity_folder) or status_code
         return status_code
 
+    def compile_pack(self):
+        return
+
+    def zipped_pack_uploader(self, path: str) -> int:
+
+        zipped_pack = Pack(path)
+        file_name = zipped_pack.path.name  # type: ignore
+
+        try:
+            zipped_pack.upload(self.client)
+
+            if self.log_verbose:
+                click.secho(f'Uploaded {FileType.PACK} - \'{os.path.basename(path)}\': successfully', fg='green')
+
+            self.successfully_uploaded_files.append((file_name, FileType.PACK.value))
+            return 0
```

Comment:
Let's change the `0` and `1` to constants with indicative names, and change it all over this page.